packages/collector/test/tracing/cloud/azure/blob/app.js

```diff
@@ -116,24 +116,19 @@ app.get('/download-buffer', async (req, res) => {
   }
 });
 
-app.get('/download-buffer-promise', async (req, res) => {
-  try {
-    await uploadDocumentToAzure();
-    blockBlobClient
-      .downloadToBuffer()
-      .then(response => {
-        fs.writeFileSync(localFilePath, response);
-        fs.unlinkSync(localFilePath);
-      })
-      .catch(error => {
-        log(`Error downloading blob: ${error.message}`);
-      });
-    await deleteDocumentFromAzure(blobName);
-    res.send();
-  } catch (e) {
-    log('Error in /download-buffer-promise:', e);
-    res.send();
-  }
+app.get('/download-buffer-promise', (req, res) => {
```

Comment:
This case was keep failing in my local, the promise was not really handled well.

Reply:
Oh yeah good catch. The logic was wrong.